src/type_sys/fender_value/mod.rs

```diff
@@ -421,7 +451,7 @@ impl ToString for FenderValue {
             FenderValue::Int(i) => i.to_string(),
             FenderValue::Float(f) => f.to_string(),
             FenderValue::Char(c) => c.to_string(),
-            FenderValue::String(s) => s.deref().clone(),
+            FenderValue::String(s) => s.iter().collect(),
```

Comment:
Is there a way to get the cached string? For example, I think `print_func` still uses `to_string` and so would copy the whole string whenever it's printed. Maybe we need to have a way to convert a type into an `impl IntoIterator<Item = char>`?